runtimes/spiritnet/src/asset_switch/mod.rs

```diff
@@ -0,0 +1,118 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use pallet_asset_switch::traits::SwitchHooks;
+use runtime_common::{AccountId, Balance};
+use xcm::{
+	v4::{Junction, Junctions, Location},
+	VersionedLocation,
+};
+
+use crate::{KiltToEKiltSwitchPallet, Runtime};
+
+const LOG_TARGET: &str = "runtime::peregrine::asset-switch::RestrictTransfersToSameUser";
```

Comment:
```suggestion
const LOG_TARGET: &str = "runtime::spiritnet::asset-switch::RestrictTransfersToSameUser";
```

Reply:
I removed the runtime prefix. I think it is obvious

Reply:
I think we need to keep it so that we can use the `-lruntime` filter for the env logger.